src/lib/stores/feedback.ts

```diff
@@ -0,0 +1,139 @@
+import { browser } from '$app/environment';
+import { VARS } from '$lib/system';
+import { writable } from 'svelte/store';
+import type { SvelteComponent } from 'svelte';
+import FeedbackGeneral from '$lib/components/feedback/feedbackGeneral.svelte';
+import FeedbackNps from '$lib/components/feedback/feedbackNPS.svelte';
+
+export type Feedback = {
+    elapsed: number;
+    visualized: number;
+    notification: boolean;
+    type: 'nps' | 'general';
+    show: boolean;
+};
+
+export type FeedbackData = {
+    message: string;
+    name?: string;
+    email?: string;
+    value?: number;
+};
+
+export type FeedbackOption = {
+    type: Feedback['type'];
+    title: string;
+    desc: string;
+    component: typeof SvelteComponent;
+};
+
+export const feedbackOptions: FeedbackOption[] = [
+    {
+        type: 'general',
+        title: 'How can we improve?',
+        desc: 'Your feedback is important to us. Please be honest and tell us what you think.',
+        component: FeedbackGeneral
+    },
+    {
+        type: 'nps',
+        title: 'How are we doing?',
+        desc: "At Appwrite, we value the feedback of our users. That means you! We'd love to hear what you think.",
+        component: FeedbackNps
+    }
+];
+
+export const selectedFeedback = writable<FeedbackOption>();
+
+function createFeedbackDataStore() {
+    const { subscribe, update } = writable<FeedbackData>({
+        message: '',
+        name: '',
+        email: '',
+        value: 0
```

Comment:
What is value for?

Reply:
We have a value for the NPS feedback (the one that measures satisfaction)